### PR TITLE
STAR-243 Spread props on checkbox input

### PR DESCRIPTION
**Pull request by allison-c**
### Purpose 🚀
_clear, one sentence description of primary purpose of this PR_

### Notes ✏️
_details of code change / secondary purposes of this PR_

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [x] MINOR (backward compatible) `@paprika/checkbox`
- [x] PATCH (bug fix) change to `@paprika/helper` `@paprika/card`

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: packages/Checkbox/src/Checkbox.js [hunk 1]
```diff
@@ -57,6 +59,9 @@ const Checkbox = props => {
 
   const checkboxId = React.useRef(uuid()).current;
   const inputRef = React.useRef(null);
+  const extendedInputProps = extractChildrenProps(children, CheckboxInputPropsCollector);
+
+  console.log(extendedInputProps);
```
Comment (nahumzs) [inline packages/Checkbox/src/Checkbox.js:64]: do you want to left the console.log?

Comment (allison-c) [reply packages/Checkbox/src/Checkbox.js:64]: No haha, thanks

File: packages/Card/stories/examples/DeluxeCard.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ export default () => {
           <Card.Title>Asset type name</Card.Title>
           <Input />
           <div style={{ width: "100%", height: "24px" }} />
-          <Checkbox>Use a display name</Checkbox>
+          <Checkbox onChange={() => {}}>Use a display name</Checkbox>
```
Comment (nahumzs) [inline packages/Card/stories/examples/DeluxeCard.js:17]: onChange shouldn't be a required value, maybe you can remove it and add a default prop with that empty function. 

There are cases where the checkbox value is provided by an upper state and might not be handle by the user.

**Comment by mikrotron** [comment]
Looks great, I like this approach of using the `<Checkbox.Input>` to capture and propagate props! Would be really lovely if you could do the same thing for the `<Radio>` too so that they both have the same API 😄

**Pull request merged**